streaming/deployment_manager_pubsub.py

```diff
@@ -0,0 +1,39 @@
+def GenerateConfig(context):
```

Comment:
Please check the latest code in iotex-etl. I split this file into two files for topics and subscriptions because the topics will be created in a different project later.